plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,14 @@
+#! bash oh-my-bash.module
+
+add_sudo() {
+  local prefix suffix
+  prefix="${READLINE_LINE:0:$READLINE_POINT}"
+  suffix="${READLINE_LINE:$READLINE_POINT}"
+  if [[ -z $prefix ]] && [[ -z $sufix ]]; then
+    prefix="$(fc -ln -1 | awk '{$1=$1};1')"
+  fi
+  READLINE_LINE="sudo $prefix$suffix"
+  ((READLINE_POINT += 5))
+}
+
+bind -x '"\e\es": add_sudo'
```

Comment:
The upstream ohmyzsh sudo seems to set up the keybindings for all of the three keymaps (corresponding to Bash's `emacs`, `vi-insert`, and `vi-command`). With the above `bind -x`, we only set the keybinding for the currently selected keymap. Can we bind all of the three as ohmyzsh sudo does? What do you think?

Reply:
I'm not familiar with vim or emacs keymap. To be honest, I'm not a used user of oh-my-bash. I needed this functionality for bash and first look was this repo, due it was not present, I thinked would be a good idea to send you a first version. If it's suitable for the framework, happy to help

Reply:
I'm not familiar with vim or emacs keymap. To be honest, I'm not a used user of oh-my-bash. I needed this functionality for bash and first look was this repo, due it was not present, I thinked would be a good idea to send you a first version. If it's suitable for the framework, happy to help

Reply:
If the reason that you would request the feature is to use a Bash version of ohmyzsh/sudo, I would expect it would work in the same way as ohmyzsh/sudo. Anyway, if the plugin has the same name as in that of ohmyzsh, I believe we should make the behavior similar to the original version.